packages/lib/sanitizeCalendarObject.ts

```diff
@@ -0,0 +1,12 @@
+import { DAVObject } from "tsdav";
+
+export const sanitizeCalendarObject = (obj: DAVObject) => {
+  return obj.data
+    .replaceAll("\r\n", "\r")
+    .replaceAll("\r", "\r\n")
+    .replaceAll(/(: \r\n|:\r\n|\r\n:|\r\n :)/gm, ":")
+    .replaceAll(/(; \r\n|;\r\n|\r\n;|\r\n ;)/gm, ";")
+    .replaceAll(/(= \r\n|=\r\n|\r\n=|\r\n =)/gm, "=");
```

Comment:
We make sure the `ICAL.parse` works with the output of this return. Currently, this seems to be fixing all potential invalid breakpoints but this can be revisited and improved if there are other edge cases we might have missed here.